### ISSUE TITLE
lr and loss function increasing?

### ISSUE THREAD

**Issue by xt17** [DeepLabCut/DeepLabCut#287]
**Your Operating system and DeepLabCut version**

Please state your operating system, env, and which version of DeepLabCut you are using.
Example: Ubuntu 16.04 LTS, with an Anaconda Env, & DeepLabCut1.x or 2.x.
Windows 1 with an Anaconda Env, & DeepLabCut2
**If applicable, please also complete the following information:**
 
**Describe the bug**
Are lr and loss supposed to be possible to increase at some point?
They increased. See the screenshot for details

**To Reproduce**
Follow the procedure of https://github.com/AlexEMG/DeepLabCut/blob/master/examples/Demo_yourowndata.ipynb
The last step was deeplabcut.train_network(path_config_file)

**Expected behavior**
They always decrease

**Screenshots**
![image](https://user-images.githubusercontent.com/50376505/57563163-73e9b580-7368-11e9-9172-2ffaa0a6ee3f.png)

**Additional context**

**Comment by MMathisLab** [comment]
It will go down, not to worry. The learning rate goes up on purpose (this is set in the pose_config file), just run it out to 200,000 or so.
